highfive/configs/_global.json

```diff
@@ -2,6 +2,7 @@
     "groups": {
         "core": ["@brson", "@nikomatsakis", "@alexcrichton", "@aturon"],
         "crates": ["@alexcrichton"],
-        "doc": ["@steveklabnik"]
+        "doc": ["@steveklabnik", "@GuillaumeGomez"],
```

Comment:
I think what you want to do here is to add "@GuillaumeGomez" to the `librustdoc` directory in `rust.json`, rather than the `doc` group. That is probably as good as you're going to get in terms of precision, I don't think you can easily get highfive to prefer you on a subdirectory.

Also nit, tab instead of spaces on the next line